target/linux/ar71xx/files/arch/mips/ath79/machtypes.h

```diff
@@ -378,6 +378,7 @@ enum ath79_mach_type {
 	ATH79_MACH_ZBT_WE1526,			/* Zbtlink ZBT-WE1526 */
 	ATH79_MACH_ZCN_1523H_2,			/* Zcomax ZCN-1523H-2-xx */
 	ATH79_MACH_ZCN_1523H_5,			/* Zcomax ZCN-1523H-5-xx */
+	ATH79_MACH_ANONABOX_PRO,                /* GainStrong MiniBox V3*/
```

Comment:
Either you want to add the "Anonabox Pro" or the "GainStrong MiniBox V3". Please decide (and add the missing space after "v3"